diagnostic.js

```diff
@@ -32,7 +41,15 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
-
+db.houses.update({
+  "members": ["Ned Stark",
+  "Arya Stark",
+  "Sansa Stark",
+  "Viserys Targaryen",
+  "Daenerys Targaryen"]
+})
+
+db.houses.update{ $push: { "houses": "members", "Jon Arryn" } }
```

Comment:
With `update`, the 1st argument is the thing we are searching by. Is your 1st argument going to return anything?

Here's 1 solution:
```js
db.houses.update(
  {name: "House Arryn"},
  {$set: { members: [ "Jon Arryn"]}}
);
```